modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -253,8 +258,21 @@
                 },
             }
             $product_config = {
+                'fenix' => { # TODO replace with fenix-nightly
```

Comment:
I'm somewhat confused by the `has_nightly_track` flag. While for `fenix` project it makes sense somehow, for `fenix-beta` and `fenix-nightly` it feels like it's adding some confusion. The former makes no sense to have a nighly channel since it's another type of shipping channel as well, while the latter is itself it 🤔

Reply:
I agree that it's bizarre - I think we're in an "in-between" state right now where we _need_ different config for each channel (e.g.: `expected_package_names`), so we need separate configs (`fenix`, `fenix-nightly`, `fenix-beta`) yet our structure right now is built around single-config-per-product.

One Fenix is stable again, I'm considering modifying this like:
```
fenix => {
    'update_google_play_strings' => false,
    'digest_algorithm' => 'SHA-256',
    'skip_...' => true,
    'channels' => [{
        'track' => 'nightly',
        'expected_package_name' => 'org.mozilla.fenix.nightly',
    }, {
        'track' => 'beta',
        'expected_package_name' => 'org.mozilla.fenix.beta',
    }]
}
```

Reply:
This looks much better, indeed! Sounds good, we can maybe file a github issue or a bug or alike to track this once the fenix is in a more stable state 👍

Reply:
Thanks :)